test/ApplicationConfiguratorTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cli for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cli/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cli/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace LaminasTest\Cli;
+
+use Laminas\Cli\ApplicationConfigurator;
+use laminas\cli\containercommandloader;
```

Comment:
Good catch, dunno what happened there but I'll change that.

Reply:
I've had psalm do this to me a few times when running `--alter`. Don't know why it does, and I've never been able to reliably reproduce the issue

Reply:
I did not use psalter here. So that was something else. Maybe an OSX issue with case insensitive filesystem...

Reply:
Okay I think thats the codestandard.
I probably used the FQCN within the code and the codestandard "fix" imported the use statement.
Since the `ContainerCommandLoader` does not really exist (due to the `class_alias` stuff which is done in the `autoload.php`), the codestandard seems to import it with lowercase characters.